classes/label.rb

```diff
@@ -4,6 +4,7 @@ class Label
 
   def initialize(params)
     @id = params[:id] || Random.rand(1..1000)
+    @name = params[:name]
```

Comment:
Hello @iamsjunaid,
- Quick question... Where did 'name' come from?

I would like to suggest you kindly to remove it...

_According to the UML diagram_
![image](https://github.com/iamsjunaid/Catalog-of-my-things/assets/67355479/327999b3-ec93-4efc-869b-e5f9afad74e8)

Reply:
that was a typo😁!!! I forgot to delete it.